core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/api/update.kt

```diff
@@ -14,6 +15,47 @@ class UpdateTests {
         df.update { col }.with { 2 } shouldBe df
     }
 
+    @DataSchema
+    interface AAndB {
```

Comment:
I need to say, that for me as a reviewer naming in this test is unclear, especially data class Update.

Could we rename it somehow to be clear from the reading or test repairing point of view

Reply:
Agreed, I'll change the names to something better :)